schedule/disperse_siblings.py

```diff
@@ -104,6 +104,8 @@ def get_due_range(cid, stability, due, desired_retention, maximum_interval):
     if due > last_review + max_ivl + 2: # +2 is just a safeguard to exclude cards that go beyond the fuzz range due to rounding 
         # don't reschedule the card to bring it within the fuzz range. Rather, create another fuzz range around the original due date.
         min_ivl, max_ivl = get_fuzz_range(due - last_review, last_elapsed_days, maximum_interval)
+        max_ivl = min(due, max_ivl) # prevent a further increase in ivl
```

Comment:
The due and ivl have different zero point. It's weird to compare them.

Reply:
Thanks for the catch.